depmap-compute/depmap_compute/context.py

```diff
@@ -36,15 +37,18 @@ def __init__(self, context):
         self.context_type = context["context_type"]
         self.expr = _encode_dots_in_vars(context["expr"])
         self.cache = {}
+        self.get_slice_data = get_slice_data
 
-    def is_match(self, entity_label):
+    def is_match(self, entity_label: str):
```

Comment:
Once I merge my changes you'll have some conflicts. I renamed `entity_label` -> `dimension_label` to keep things a little more consistent.